spec/requests/users_controller_spec.rb

```diff
@@ -31,6 +33,24 @@
         post api_v1_users_path, params: valid_user_params, headers: headers
         expect(User.find_by(email: valid_user_params[:user][:email]).role).to eq(role)
       end
+
+      context 'User language' do
+        it 'Persists the user language in the user record' do
+          expect { post api_v1_users_path, params: valid_user_params, headers: }.to change(User, :count).from(0).to(1)
+          expect(response).to have_http_status(:created)
+          expect(User.find_by(email: valid_user_params[:user][:email]).language).to eq('en')
```

Comment:
We can cut down on the code duplication a bit here

```
post api_v1_users_path, params: valid_user_params, headers:
expect(User.find_by(email: valid_user_params[:user][:email]).language).to eq('en')
```

Reply:
But that means that we won't be ensuring an adequate response code and that the user in the database may not be created by the `post api_v1_users_path` or is it normal to be omitted?

Reply:
Here's what I thought when developed the tests:
>I wanted to ensure that setting the language won't prevent the creation of a user (that's why I was testing the creation) and the response should be as expected for the client app (that's why I was testing the response status code).